examples/heatTransfer/read/heatRead_adios2.cpp

```diff
@@ -109,7 +111,7 @@ int main(int argc, char *argv[])
     double *T = new double[vT->m_AvailableSteps * readsize[0] * readsize[1]];
```

Comment:
The problem is that the printData function is used by heatRead_adios1 and heatRead_adios2 at the same time. If I change the printData to using adios2::Dims then I have to add two sets of these variables in heatRead_adios1 instead of adding them into heatRead_adios2.